### PR TITLE
Add support for new In expressions

### PR DESCRIPTION
**Pull request by frank400**

### CODE REVIEW

**Comment by projjal** [comment]
Hi @frank400 Can you add unit tests for this. You can look at testInOptimization method in TestNativeFunctions.java for reference.

File: sabot/kernel/src/main/java/com/dremio/sabot/op/llvm/GandivaExpressionBuilder.java [hunk 1]
```diff
@@ -227,6 +231,26 @@ public TreeNode visitUnknown(LogicalExpression e, Void value) {
...
+        for (LogicalExpression constant: in.getConstants()){
+          if (constant instanceof DecimalExpression){
+            decimalValues.add(((DecimalExpression) constant).getDecimal());
+          } else if (constant instanceof FunctionHolderExpression){
+            LogicalExpression val = ((FunctionHolderExpression) constant).args.get(0);
+            if (val instanceof DecimalExpression){
+              DecimalExpression expr = (DecimalExpression) val;
+              decimalValues.add(expr.getDecimal());
+            }else if (!(val instanceof TypedNullConstant)) {
+              throw new UnsupportedOperationException("Supports only int, decimal, float, " +
```
Comment (projjal) [inline sabot/kernel/src/main/java/com/dremio/sabot/op/llvm/GandivaExpressionBuilder.java:245]: float, double should not be in this patch

**Comment by projjal** [comment]
You need to add decimal in this list https://github.com/s1mbi0se/dremio-oss/pull/2/files#diff-54967b8ab1983fa4a269a5502b0f10fec9afd33b03fbd2a8556bc8785d9b9436R72

**Comment by jpedroantunes** [comment]
ALREADY SENT TO PROJJAL

**Pull request closed without merging**